### PR TITLE
Show selection in transparency for non-visible objects

### PR DESCRIPTION
**Pull request by martinRenou**
Fix https://github.com/jupytercad/JupyterCAD/issues/562

https://github.com/user-attachments/assets/e3df0d47-e241-4f66-a61c-df4e3c5f9f78

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Awesome! I noticed that when moving the element, it completely disappears.

File: packages/base/src/3dview/helpers.ts [hunk 1]
```diff
@@ -190,6 +190,9 @@ export function buildShape(options: {
   const meshGroup = new THREE.Group();
   meshGroup.name = `${objName}-group`;
   meshGroup.visible = visible;
+  meshGroup.userData = {
```
[the remark this answers is not in the document]
Comment (martinRenou) [reply packages/base/src/3dview/helpers.ts:193]: `userData.visible` is the jcad object visibility
`visible` is the mesh group visibility (can be visible even though the jcad object is not)

I'm thinking we may want to keep a reference to the original jcad object instead `userData.jcObject`, what do you think?

[the remark this answers is not in the document]
Comment (martinRenou) [reply packages/base/src/3dview/helpers.ts:193]: I just pushed a commit storing `userData.jcObject`, that should make the code clearer

**Comment by arjxn-py** [comment]
This looks super awesome 🎉🤩

File: packages/base/src/3dview/helpers.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (trungleduc) [reply packages/base/src/3dview/helpers.ts:193]: Thanks!

**Review by trungleduc — APPROVED** [review]

**Pull request merged**